pkg/upgrade/upgrade.go

```diff
@@ -104,6 +104,16 @@ func ManagedInstance(ctx context.Context, client client.Client, jaeger v1.Jaeger
 		return jaeger, nil
 	}
 
+	if currentSemVersion.GreaterThan(latestSemVersion) {
+		// This jaeger instance has a version greater than the latest know version of the operator
```

Comment:
Remove `know` from here, and also the warning message below.

Reply:
+1